### ISSUE TITLE
[ Feature Request ] Add WebP/JPG Conversion to all uploaded images

### ISSUE THREAD

**Issue by xdedeone** [andrii-kryvoviaz/slink#23]
Hi, first of all, good job. Im looking for a modern image uploader like this.
I suggest to add a option to automatic image conversion/compression for all uploaded images to a specific image format. This for images with large format, to minimize disk space usage in a self hosted server.

**Comment by andrii-kryvoviaz** [comment]
Definitely, I like this idea. It could be just another setting under "admin panel" (when it is released), where it would be possible to select the format and/or quality in case of jpg to convert to. I am open to any suggestions.

In terms of an app architecture, I want to do it in the background in separate worker in order to not slow down the uploads and do not introuduce extra load spikes. Will see, maybe it would be possible to use GPU acceleration for such conversions.
